packages/actions/templates/app-build/template.yml

```diff
@@ -0,0 +1,77 @@
+##################################################################################
+#         Configuration
+##################################################################################
+env:
+  DEPLOYMENT_NAME: "%{DEPLOYMENT_NAME}"
+  DEPLOYMENT_PRIVATE_KEY: ${{secrets.DEPLOYMENT_PRIVATE_KEY}}
+  FIREBASE_CONFIG: ${{secrets.FIREBASE_CONFIG}}
+
+##################################################################################
+#         Main Code
+##################################################################################
```

Comment:
Would be good to add a `name`, as with the "Deploy - Firebase" action, so that it gets named in Github Actions 
<img width="328" alt="Screenshot 2023-06-27 at 10 37 56" src="https://github.com/IDEMSInternational/parenting-app-ui/assets/64838927/0e19fa09-14e5-4e8d-bd61-16c5af89bab1">